internal/pgbackrest/reconcile.go

```diff
@@ -60,6 +60,25 @@ func AddRepoVolumesToPod(postgresCluster *v1beta1.PostgresCluster, template *cor
 			},
 		})
 
+		var initContainerFound bool
+		var index int
+		for index = range template.Spec.InitContainers {
+			if template.Spec.InitContainers[index].Name == naming.ContainerPGBackRestLogDirInit {
+				initContainerFound = true
+				break
+			}
+		}
+		if !initContainerFound {
+			return errors.Errorf(
```

Comment:
I think the error is worth keeping, if the init container isn't found, there will be problems down the line since the proper directory is not created. I've updated the test to check for the relevant error.